en/user_guide/keyword_management.rst

```diff
@@ -0,0 +1,12 @@
+Keyword management
+==================
+
+Open Orchestra allows to attach keyword to content or media. They describe the content and allow to manage it.
+In the keyword panel, you can create or destroy a keyword.
```

Comment:
Development question : does the fact to destroy a keyword in the list, remove it from objects using it ?